.github/workflows/commits.yml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Conventional commit check
+on: [push]
+jobs:
+  commit_messages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Load scopes from file
+        id: load_scopes
+        run: |
+          scopes_file=".github/.scopes.txt"
+          scopes=$(awk 'NR == 1 { printf $1 } NR > 1 { printf "|" $1 }' $scopes_file)
+          echo "::set-output name=scopes::$scopes"
+
+      - name: Conventional commit check
```

Comment:
I think it would be better to keep using the pre-made step for checking the conventional commit as a whole. Hence, this step would only check the scopes.